src/components/Viewer/InformationPanel/About/About.tsx

```diff
@@ -56,6 +59,9 @@ const About: React.FC = () => {
         <SeeAlso
           seeAlso={seeAlso as unknown as PrimitivesExternalWebResource[]}
         />
+        <Rendering
+          rendering={rendering as unknown as PrimitivesExternalWebResource[]}
+        />
```

Comment:
I wonder of this should have a `target="_blank"`. Might be assumptive to force a download but it could be beneficial for the user to not be navigated away here. This might be true for all of these linking properties, honestly.

Reply:
Added this here, but I agree it should probably be everywhere.